templates/telecommunications/index.html

```diff
@@ -0,0 +1,207 @@
+  <div class="row">
+    <div class="col-8">
+      <h2>A telco-grade managed cloud</h2>
+      <p>Canonical&rsquo;s Foundation Cloud Build for Telcos is a pre-tested, pre-integrated Openstack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about Canonical&rsquo;s Foundation Cloud Build</span>&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Smart edge: Internet of Things, switches and CPEs</h2>
+      <p>Ubuntu is the Operating System of choice for the smart edge. From software-defined top of the rack switch to enterprise CPE, from home IoT gateways to digital signage, Canonical offers secure OS and appstore solutions to maximise post sales revenue from all your smart devices.</p>
+      <p><a href="/internet-of-things">Find out more<span class="u-off-screen"> about smart edge</span>&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<div class="p-strip--accent is-deep">
+  <div class="row">
+    <div class="col-12">
+      <blockquote class="p-pull-quote--accent">
+        <p>We&rsquo;re reinventing how we scale by becoming simpler and modular, similar to how applications have evolved in cloud data centers. Open source and OpenStack innovations represent a unique opportunity to meet these requirements and Canonical&rsquo;s cloud and open source expertise make them a good choice for AT&T.</p>
+        <cite class="p-pull-quote__citation">Toby Ford, Assistant Vice President of Cloud Technology, Strategy and Planning at AT&T</cite>
+      </blockquote>
+    </div>
+  </div>
+</div>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-12">
+        <h2 class="p-muted-heading u-align--center">A selection of our telecommunications clients</h2>
+        <ul class="p-inline-images">
+          <li class="p-inline-images__item">
+            <img src="{{ ASSET_SERVER_URL }}25de1877-Tele2.svg" alt="Tele2" />
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}03a06060-logo-deutschetelekom.svg" width="242" alt="Deutshe Telekom logo">
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}b6e008be-Colt-Logo.svg" alt="Colt logo" />
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}481063cd-pccw.svg" alt="PCCW logo" />
+          </li>
+          <li class="p-inline-images__item">
+            <img class="p-inline-images__logo" src="{{ ASSET_SERVER_URL }}071c255b-AT%26T_logo.svg" alt="AT&amp;T logo" />
```

Comment:
You don't need to encode ampersands in the alt text

Reply:
Done